tests/sentry/tasks/test_llm_issue_detection.py

```diff
@@ -20,3 +57,116 @@ def test_run_detection_dispatches_sub_tasks(self, mock_delay):
+        mock_seer_api.return_value = mock_response
+
+        with pytest.raises(LLMIssueDetectionError) as exc_info:
+            detect_llm_issues_for_project(self.project.id)
+
+        exception = exc_info.value
+        assert exception.message == "Seer response parsing error"
+        assert exception.status == 200
+        assert exception.project_id == self.project.id
+        assert exception.trace_id is not None
+        assert exception.response_data == "invalid json"
+        assert exception.error_message is not None
+        assert "Expecting value" in exception.error_message
+
+    def test_detect_llm_issues_invalid_schema(self, mock_seer_api):
+        """Test Pydantic validation error handling."""
+        self._create_test_transaction_data()
+        mock_response = Mock(spec=BaseHTTPResponse)
+        mock_response.status = 200
+        mock_response.data = b'{"wrong_field": "value"}'
+        mock_response.json.return_value = {"wrong_field": "value"}
+        mock_seer_api.return_value = mock_response
+
+        with pytest.raises(LLMIssueDetectionError) as exc_info:
+            detect_llm_issues_for_project(self.project.id)
+
+        exception = exc_info.value
+        assert exception.message == "Seer response parsing error"
+        assert exception.status == 200
+        assert exception.project_id == self.project.id
+        assert exception.trace_id is not None
+        assert exception.response_data == '{"wrong_field": "value"}'
+        assert exception.error_message is not None
+        assert "field required" in exception.error_message
+
+    @patch("sentry.tasks.llm_issue_detection.logger")
+    def test_detect_llm_issues_success_with_logging(self, mock_logger, mock_seer_api):
+        """Test successful issue detection with proper logging."""
+        self._create_test_transaction_data()
+        mock_response = Mock(spec=BaseHTTPResponse)
+        mock_response.status = 200
+        mock_response.data = b'{"issues": [{"title": "Test Issue", "explanation": "Test explanation", "impact": "High", "evidence": "Test evidence"}]}'
+        mock_response.json.return_value = {
+            "issues": [
+                {
+                    "title": "Test Issue",
+                    "explanation": "Test explanation",
+                    "impact": "High",
+                    "evidence": "Test evidence",
+                }
+            ]
+        }
+        mock_seer_api.return_value = mock_response
+
+        detect_llm_issues_for_project(self.project.id)
+
+        success_log_calls = [
+            call
+            for call in mock_logger.info.call_args_list
+            if "Seer issue detection success" in call[0]
+        ]
+        assert len(success_log_calls) > 0
+        success_call = success_log_calls[0]
+        log_extra = success_call[1]["extra"]
+        assert log_extra["num_issues"] == 1
+        assert "trace_id" in log_extra
+
+    def test_detect_llm_issues_no_transactions(self, mock_seer_api):
```

Comment:
Do we need all these tests? there are quite a few validation ones, but im concerned they outnumber ones that actually test behavior which could make this file less readable